lib/cocoapods/xcode/xcframework.rb

```diff
@@ -5,6 +5,10 @@
 module Pod
   module Xcode
     class XCFramework
+      # @return [String] target_name the target name this XCFramework belongs to
+      #
+      attr_reader :target_name
```

Comment:
do we need to store this here? it's odd that what was previously agnostic to the context now depends on what target provides it. if we only need this for the embed frameworks script can be provide that info some other way?

Reply:
thats a good point let me see what I can do.

Reply:
we also need it for the copy-xcframeworks script or as you suggested use the `config` there as well.